py_gtfs_rt_ingestion/tests/test_gtfs_rt_converter.py

```diff
@@ -16,7 +17,9 @@ def test_bad_conversion_local() -> None:
     be added to the error files list
     """
     # dummy config to avoid mypy errors
-    converter = GtfsRtConverter(config_type=ConfigType.RT_ALERTS)
+    converter = GtfsRtConverter(
+        config_type=ConfigType.RT_ALERTS, metadata_queue=Queue()
+    )
```

Comment:
i've changed these tests so many times. next time, i should make a generic one to make these syntax changes easier.